d2layouts/d2grid/layout.go

```diff
@@ -126,6 +122,34 @@ func withoutGridDiagrams(ctx context.Context, g *d2graph.Graph, layout d2graph.L
 			}
 			if obj.LabelDimensions.Height != 0 {
 				labelHeight := float64(obj.LabelDimensions.Height) + 2*label.PADDING
+
+				{
```

Comment:
what's the conflicting variables that require this to be in its own scope?

Reply:
this is just for clarity to limit these to its own scope